lib/graphql/execution/execute.rb

```diff
@@ -76,8 +76,8 @@ def resolve_field(owner, selection, parent_type, field, object, query_ctx)
             selection: selection,
           )
 
-          arguments = query.arguments_for(selection, field)
```

Comment:
exceptions raised here weren't being handled which is how the ExecutionError ends up in the query results as described in #798.